Library/Formula/gtk+3.rb

```diff
@@ -22,9 +25,26 @@ class Gtkx3 < Formula
   depends_on "pango"
   depends_on "glib"
 
+  if build.with?("icons")
+    # Adwaita icon theme dependencies
+    depends_on "gettext" => :build
+    depends_on "icon-naming-utils" => :build
+    depends_on "intltool" => :build
+    depends_on "librsvg"
```

Comment:
I'm not quite sure of the best way to handle this. `gtk+3` is a relatively popular option in `librsvg` from what I've seen, so it'd probably upset people if we killed it off. I guess we could do something like:

```
depends_on "librsvg" => "without-gtk+3"
```

Or such, if that'd work.

CC @Homebrew/owners for viewpoints on what do about the circular dep potential here.